src/Apps/FeatureAKG/routes.tsx

```diff
@@ -0,0 +1,46 @@
+import loadable from "@loadable/component"
+import { RouteConfig } from "found"
+import { graphql } from "react-relay"
+
+export const routes: RouteConfig[] = [
+  {
+    path: "/campaign/art-keeps-going",
+    getComponent: () => loadable(() => import("./FeatureApp")),
+    query: graphql`
+      query routes_FeatureQuery(
+        $articleIDs: [String]!
+        $selectedWorksSetID: String!
+        $collectionRailItemIDs: [String!]
+        $auctionRailItemIDs: [String!]
+        $fairRailItemIDs: [String!]
+      ) {
+        viewer {
+          ...FeatureApp_viewer
+            @arguments(
+              articleIDs: $articleIDs
+              selectedWorksSetID: $selectedWorksSetID
+              collectionRailItemIDs: $collectionRailItemIDs
+              auctionRailItemIDs: $auctionRailItemIDs
+              fairRailItemIDs: $fairRailItemIDs
+            )
+        }
+      }
+    `,
+    prepareVariables: (_params, props) => {
+      const data = props.context.injectedData
+      const collectionIDs = data?.browse?.collections_rail?.items
+      const fairIDs = data?.browse?.fairs_rail?.items
+      const auctionIDs = data?.browse?.auctions_rail?.items
+
+      return {
+        articleIDs: data?.editorial?.article_ids,
+        selectedWorksSetID: data?.selected_works?.set_id,
+        collectionRailItemIDs:
+          collectionIDs.length && collectionIDs.map(item => item.id),
```

Comment:
Ok that this as well as the other `IDs` props could be falsy? Better to set at the definition and use the new `??` [nullish coalescing](https://levelup.gitconnected.com/understand-the-nullish-coalescing-operator-coming-in-es2020-2d1c6df1765f) operator: 

```ts
const collectionIDs = data?.browse?.collections_rail?.items.map(item => item.id) ?? []
const fairIDs = ...
const auctionIDs = ...
```